### PR TITLE
Setting screen

### PR DESCRIPTION
**Pull request by chuongmd-1771**
### Issue Link :link:

https://edu-redmine.sun-asterisk.vn/issues/10029
https://edu-redmine.sun-asterisk.vn/issues/10030

### CODE REVIEW

**Comment by chuongmd-1771** [comment]
ready

**Comment by chuongmd-1771** [comment]
ready

**Comment by chuongmd-1771** [comment]
ready

File: Petty/Data/Model/PetImages.swift [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+//  Copyright © 2019 Sun-Asterisk. All rights reserved.
+//
+
+import UIKit
+
+struct PetImages {
+    var petImage: UIImageView?
+    var petName: String?
+    var petStatus: String?
+
```
Comment (tienpx3010) [inline Petty/Data/Model/PetImages.swift]: Thừa dòng

File: Petty/Widgets/CustomCell/homeNewFeedCell.swift [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import Reusable
+
+final class homeNewFeedCell: UITableViewCell, NibReusable {
+    @IBOutlet weak var avatarImage: UIImageView!
+    @IBOutlet weak var petNameTextField: UITextField!
+    @IBOutlet weak var petStatusTextField: UITextField!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
```
Comment (tienpx3010) [inline Petty/Widgets/CustomCell/homeNewFeedCell.swift]: Bỏ comment

**Comment by chuongmd-1771** [comment]
ready

**Review by tienpx3010 — APPROVED** [review]

**Pull request merged**